src/antsibull_docs/cli/antsibull_docs.py

```diff
@@ -277,6 +277,10 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
                                          ' release of Ansible')
     devel_parser.add_argument('--pieces-file', default=DEFAULT_PIECES_FILE,
                               help='File containing a list of collections to include')
+    devel_parser.add_argument('--use-current-ansible-core', action='store_true',
```

Comment:
IMO, something like `--use-installed-ansible-core` would be better. `current` seems confusing here.

Reply:
Renamed in ab51201a32d2cfa3bd911afeab8ee7a9b82aa26b.